export/src/export.js

```diff
@@ -12,6 +6,7 @@ const { normalize } = require('./helpers/filename')
 const { getArticleById, getVersionById, getCorpusById } = require('./graphql')
 
 const canonicalBaseUrl = process.env.EXPORT_CANONICAL_BASE_URL
+const exportEndpoint = process.env.SNOWPACK_PUBLIC_PANDOC_EXPORT_ENDPOINT || 'http://127.0.0.1:3080'
```

Comment:
I wonder if that's a safe option to set the default fallback URL within the code, isn't it possible to override that with an env var on a per-env basis? (I know nothing about how env var are dealt with so it might be a dumb remark)

Reply:
You're right, I've added some config typing with the `convict` module. It will fail at boot-time rather than at run-time.